src/tags/tags_test.js

```diff
@@ -61,13 +65,23 @@ describe('tags', function () {
         tags.start('haml', 'css').should.equal('-# inject:css');
         tags.start('haml', 'html').should.equal('-# inject:html');
         tags.start('haml', 'js').should.equal('-# inject:js');
+        tags.start('haml', 'less').should.equal('-# inject:less');
         tags.start('haml', 'UNKNOWN').should.equal('-# inject:UNKNOWN');
       });
 
+      it('should return less comment tag for less files', function () {
+        tags.start('less', 'css').should.equal('/* inject:css */');
+        tags.start('less', 'html').should.equal('/* inject:html */');
```

Comment:
should i test options that should not happen. less does not support inserting html files.